extending-the-rest-api/routes-and-endpoints.md

```diff
@@ -516,7 +516,7 @@ function prefix_data_arg_validate_callback( $value, $request, $param ) {
  * @param  mixed            $value   Value of the 'filter' argument.
  * @param  WP_REST_Request  $request The current request object.
  * @param  string           $param   Key of the parameter. In this case it is 'filter'.
- * @return WP_Error|boolean
+ * @return string
```

Comment:
Thanks for the catch on the `boolean` -> `string` issue!
```suggestion
 * @return WP_Error|string
```
I do think we need to document the option to return a `WP_Error`: This particular function does not return one, but it is valid, and there are cases where it can be useful to return an error from a sanitizer.